grpc-proto/proto/query.proto

```diff
@@ -188,11 +190,13 @@ message QueryParameters {
   // example when fetching multiple pages for the same query).
   bool skip_metadata = 6;
 
-  // The query timestamp (in microseconds). This is used to order mutations on the server. If no
-  // timestamp is provided, the server will assign one based on the time it received the query.
+  // The query timestamp (in microseconds). This is used to order mutations on the server.
+  // If unset, the server will assign one based on the time it received the query.
   google.protobuf.Int64Value timestamp = 7;
 
   // The serial consistency level (if the query is a lightweight transaction).
+  // If unset, a default will be used but is unspecified and might depend of the Persistence
+  // implementation.
```

Comment:
Again from the `Parameters` javadoc.
I think it would be better to force a deterministic default in the service.

Reply:
#1157